community/modules/scheduler/schedmd-slurm-gcp-v6-controller/slurm_files.tf

```diff
@@ -84,10 +84,12 @@ locals {
     filename = "ghpc_startup.sh"
     content  = var.compute_startup_script
   }]
+  nodeset_startup_scripts = {
+  for ns in var.nodeset : ns.nodeset_name => ns.startup_script }
 }
 
 module "slurm_files" {
-  source = "github.com/GoogleCloudPlatform/slurm-gcp.git//terraform/slurm_cluster/modules/slurm_files?ref=6.4.2&depth=1"
+  source = "github.com/GoogleCloudPlatform/slurm-gcp.git//terraform/slurm_cluster/modules/slurm_files?ref=partition_script&depth=1"
```

Comment:
I affirm that this PR has the functionality we are looking for. 

Send it back to me when you have a tagged version in the Slurm repo.
PS. In final version, we will want to roll all references.